apps/scale.go

```
@@ -140,6 +147,41 @@ func Scale(app string, sync bool, types []string) error {
 	return nil
 }
 
+func formatContainerTypesError(c *scalingo.Client, app string) error {
+	containerTypes, _ := c.AppsPs(app)
```

Woot? Ignoring the error? Seems like a terrible idea, don't you think?